graylog-storage-elasticsearch7/src/main/java/org/graylog/storage/elasticsearch7/views/ElasticsearchBackend.java

```diff
@@ -167,6 +167,13 @@ public ESGeneratedQueryContext generate(SearchJob job, Query query, SearchConfig
                 searchTypeOverrides.must(normalizedSearchTypeQuery);
             });
 
+            usedSearchFiltersToQueryStringsMapper.map(searchType.filters())
+                    .forEach(searchFilterQueryString -> {
```

Comment:
same as above, more map steps maybe?

Reply:
Good idea, I've improved it, let me know if you like to more. :)